### PR TITLE
Changing voq tests to use sonic-db-cli instead of redis-cli

### PR DESCRIPTION
**Pull request by mannytaheri**
### Description of PR
Changing voq tests to use sonic-db-cli instead of redis-cli
redis-cli is getting obsoloted and is broken in latest github for multi-asic

### Type of change
- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Approach
Changing voq tests to use sonic-db-cli instead of redis-cli

#### How did you do it?
Changed voq tests to use sonic-db-cli instead of redis-cli

#### How did you verify/test it?
Tested it against a VoQ chassis

### CODE REVIEW

File: tests/common/helpers/sonic_db.py [hunk 1]
```diff
@@ -6,27 +6,24 @@
...
         self.host = host
         self.database = database
-        self.pid = pid
-        self.ip = None
 
     def _cli_prefix(self):
-        """Builds opening of redis CLI command for other methods."""
-        # return "docker exec -i {docker} redis-cli -p {pid} --raw -n {db} ".format(
+        """Builds opening of sonic-db-cli command for other methods."""
+        # return "sonic-db-cli -p {pid} --raw -n {db} ".format(
```
Comment (wangxin) [inline tests/common/helpers/sonic_db.py:24]: Please remove the unrelated and misleading comments.

**Review by wangxin — APPROVED** [review]
LGTM except one minor issue of comments in code.

**Comment by wangxin** [comment]
@mannytaheri Can you address the review comments and resolve the conflicts?

File: tests/common/devices/sonic_asic.py [hunk 2]
```diff
@@ -142,6 +142,13 @@ def show_ip_interface(self, *module_args, **complex_args):
         complex_args['namespace'] = self.namespace
         return self.sonichost.show_ip_interface(*module_args, **complex_args)
 
+    def run_sonic_db_cli_cmd(self, sonic_db_cmd):
+        if self.namespace != DEFAULT_NAMESPACE:
```
[the remark this answers is not in the document]
Comment (mannytaheri) [reply tests/common/devices/sonic_asic.py]: Fixed in last commit

File: tests/common/helpers/sonic_db.py (hunk 1 above)
Comment (mannytaheri) [reply tests/common/helpers/sonic_db.py:24]: Fixed in last commit

File: tests/voq/test_voq_intfs.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
+
```
[the remark this answers is not in the document]
Comment (mannytaheri) [reply tests/voq/test_voq_intfs.py]: Removed in the latest commit

File: tests/common/helpers/sonic_db.py [hunk 4]
```diff
@@ -6,27 +6,24 @@
...
         #     docker=self.docker, db=self.database, pid=self.pid)
```
Comment (smaheshm) [inline tests/common/helpers/sonic_db.py:25]: can you also remove the commented code.

**Comment by smaheshm** [comment]
@mannytaheri Can you address this issue

```
ImportError while importing test module '/var/src/s/tests/decap/test_decap.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
decap/test_decap.py:16: in <module>
    from tests.common.fixtures.fib_utils import fib_info_files
common/fixtures/fib_utils.py:9: in <module>
    from tests.common.helpers.redis import VoqDbCli
E   ImportError: No module named redis
_______________________ ERROR collecting fib/test_fib.py _______________________
ImportError while importing test module '/var/src/s/tests/fib/test_fib.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
fib/test_fib.py:19: in <module>
    from tests.common.fixtures.fib_utils import fib_info_files_per_function
common/fixtures/fib_utils.py:9: in <module>
    from tests.common.helpers.redis import VoqDbCli
E   ImportError: No module named redis
```

**Review by smaheshm — APPROVED** [review]
looks good.

**Review by smaheshm — APPROVED** [review]

**Comment by sanmalho-git** [comment]
@arlakshm - the changes requested by you are taken care of. Could you please review again and if approved, please merge this PR.

**Review by arlakshm — APPROVED** [review]

**Comment by arlakshm** [comment]
please resolve the conflicts

**Comment by sanmalho-git** [comment]
@arlakshm - have rebased and resolved the conflicts. Please merge

**Pull request merged**